charts/datadog/templates/daemonset.yaml

```
@@ -9,6 +9,7 @@ metadata:
   labels:
 {{ include "datadog.labels" . | indent 4 }}
     app.kubernetes.io/component: agent
+    admission.datadoghq.com/enabled: false
```

```suggestion
    admission.datadoghq.com/enabled: "false"
```

the `false` need to be considered as a string.